tests/drivers/eeprom/boards/native_posix.conf

```diff
@@ -3,6 +3,5 @@
 
 CONFIG_EMUL=y
 CONFIG_I2C=y
-CONFIG_EEPROM_AT24=y
```

Comment:
Why not removing `CONFIG_I2C=y` here as well?

Reply:
done.